README.md

```diff
@@ -94,6 +96,33 @@ it:
 Prone Support. Alternatively reference this project's `self-check` profile
 definition. -->
 
+#### Gradle (Groovy)
+
+- Add `net.ltgt.errorprone` plugin to your `build.gradle` file with one or 
+  more Error Prone Support modules:
+
+   ```groovy
+   plugins {
+       // The gradle plugin is an external contribution.
```

Comment:
I got these lines from [this page ](https://errorprone.info/docs/installation).

Reply:
In the Maven section above we link to that page, and then only document the "delta". Perhaps we can directly link to the [gradle-errorprone-plugin](https://github.com/tbroyer/gradle-errorprone-plugin), and then only document what further changes are necessary?

Reply:
I am working on this, will push a suggestion ~soon.

Reply:
It looks better already.